examples/basic-async/basic-async.c

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2021, Intel Corporation */
+
+#include <stdlib.h>
+#include <string.h>
+#include "libminiasync.h"
+
+int
+main(void)
```

Comment:
Example for the user should be documented (README) or commented properly in the code.